playbooks/roles/forum/templates/forum-watchdog.sh.j2

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+{% if ! devstack %}
```

Comment:
I'm getting `"AnsibleError: template error while templating string: unexpected char u'!' at 19. String: #!/bin/bash\n\n{% if ! devstack %}\n  tail -f {{ supervisor_log_dir }}/forum-stderr.log |\n  grep --line-buffered 'Mongo::Error::OperationFailure - not master and slaveOk=false (13435)' |\n  while read;\n  do\n    dt=`date '+%d/%m/%Y %H:%M:%S'`\n    echo \"[$dt] Forum Failure detected - Restarting forum service...\"\n    {{ supervisor_ctl }} restart forum\n  done\n{ % else %}\n  # On devstack just run a dummy loop to keep supervisor from trying to\n  # restart service\n  while true; do sleep 1d; done\n{% endif %}\n"` here.

Reply:
Yes